log-manager/src/main/java/io/airlift/log/Logging.java

```diff
@@ -81,15 +81,22 @@ public static synchronized Logging initialize()
 
     private Logging()
     {
-        ROOT.setLevel(Level.INFO.toJulLevel());
```

Comment:
Why remove this?

Reply:
It became redundant with LogManager.reset() which does the same. See its resetLogger: (defaultLevel is Level.INFO)
```
        if (name != null && name.isEmpty()) {
            // This is the root logger.
            logger.setLevel(defaultLevel);
```